contributing/language-translations.md

```diff
@@ -1,4 +1,4 @@
-(translations)=
+(language-translations)=
```

Comment:
You need to update all references to this target if you change it. From the warnings in the rtd build it looks like this one is only used once at https://github.com/pymc-devs/pymc-data-umbrella/blob/main/contributing/index.md?plain=1#L74.